docs/.vuepress/public/presets/toml/parrot-os.toml

```diff
@@ -0,0 +1,53 @@
+format = '''
+[┌](fg:red)$status$os$hostname$time$directory$git_branch$git_status
+[└──╼](fg:red)$username$character
+'''
+
+[status]
+style = 'fg:white'
+symbol = '✗'
+format = '[\[](fg:red)[$symbol]($style)[\]](fg:red)'
+disabled = false
+
+[character]
+success_symbol = '[\$](fg:yellow)'
+error_symbol = '[\$](fg:yellow)'
+vimcmd_symbol = '[\$](fg:yellow)'
+vimcmd_replace_one_symbol = '[\$](fg:yellow)'
+vimcmd_replace_symbol = '[\$](fg:yellow)'
+vimcmd_visual_symbol = '[\$](fg:yellow)'
```

Comment:
I'm uncertain if it is a good idea to use the same symbol for all different character values. In my opinion, that could lead to confused users.

I would prefer if someone from the maintainers would voice their opinion.

Reply:
This is a very valid point. It would be great if there was another solution, maybe there is one I don't know about?

Reply:
Use different characters and colors?

Reply:
What I meant is, it would be great to have something like a wrapper, so that I can specify the surrounding for all variants of character. I wanted do it as much ParrotOS'ish as possible, that's why I chose exactly `$` and these colours.
Am I misunderstanding your comment?

Reply:
I removed the usage of `character` now since it's not necessary to imitate ParrotOS' shell style.